pkgs/build-support/docker/default.nix

```diff
@@ -82,7 +82,7 @@ rec {
     export PATH=${shadow}/bin:$PATH
     mkdir -p /etc/pam.d
     if [[ ! -f /etc/passwd ]]; then
-      echo "root:x:0:0::/root:/bin/sh" > /etc/passwd
+      echo "root:x:0:0::/root:${stdenv.shell}" > /etc/passwd
```

Comment:
is this save to use? I think there is a reason why on nixos the root login shell points to `/run/current-system/sw/bin/bash`.